src/scanner.h

```diff
@@ -216,7 +216,6 @@ typedef UInt            TypSymbolSet;
 */
```

Comment:
`We only fill 'Value' up to SAFE_VALUE_SIZE normally.` above should then also be changed to `We only fill 'Value' up to MAX_VALUE_LEN normally.`

Reply:
Thanks for looking very carefully, I appreciate it! But actually, that `SAFE_VALUE_SIZE` is not there anymore, as it has been removed in master already. But I'll rebase this PR to "prove" it ;-).

Reply:
Haha. OK :)